Valet/VALValet.m

```diff
@@ -18,7 +18,6 @@
 //  limitations under the License.
 //
 
-#import "VALValet.h"
```

Comment:
`VALValet_Protected.h` imports `VALValet.h`, which meant this import caused duplicate symbols.